### DISCUSSION TITLE
Can we embed a gist or other github file in the docs?

### DISCUSSION THREAD

**Discussion by SteveRosam** [squidfunk/mkdocs-material#5660] [Q&A] [answered]
To make it easier for users, I want them to be able to easily download the code rather than just copy / paste it into a blank text file.

**Comment by squidfunk** [comment, marked as answer]
Sure, just use the standard embed functionality that is offered by GitHub. For demonstration, add this to your Markdown file:

``` html
<script src="https://gist.github.com/squidfunk/ec27cf48906f188916e4970e7930c405.js"></script>
```

Result:

<img width="714" alt="Bildschirmfoto 2023-06-30 um 14 30 41" src="https://github.com/squidfunk/mkdocs-material/assets/932156/f0e87af2-cf1b-46d6-8bc2-e5e1cb8fe7df">
